Lib/defconAppKit/tools/drawing.py

```diff
@@ -147,7 +147,7 @@ def drawTextAtPoint(text, pt, scale, attributes={}, xAlign="left", yAlign="botto
 
 # Vertical Metrics
 
-def drawFontVerticalMetrics(glyph, scale, rect, drawLines=True, drawText=True, color=None, backgroundColor=None):
+def drawFontVerticalMetrics(glyph, scale, rect, drawLines=True, drawText=True, color=None, backgroundColor=None, flipped=False):
```

Comment:
if posted a pull pull request that fixes most of these issues
and there are more to come...

gr Frederik
www.typemytype.com

> On 13 Jul 2016, at 14:40, Miguel Sousa <notifications@github.com> wrote:
> 
> In Lib/defconAppKit/tools/drawing.py <https://github.com/typesupply/defconAppKit/pull/15#discussion_r70616761>:
> 
> > @@ -147,7 +147,7 @@ def drawTextAtPoint(text, pt, scale, attributes={}, xAlign="left", yAlign="botto
> >  
> >  # Vertical Metrics
> >  
> > -def drawFontVerticalMetrics(glyph, scale, rect, drawLines=True, drawText=True, color=None, backgroundColor=None):
> > +def drawFontVerticalMetrics(glyph, scale, rect, drawLines=True, drawText=True, color=None, backgroundColor=None, flipped=False):
> This is to address this traceback:
> 
> Traceback (most recent call last):
>   File "defconAppKit/controls/glyphView.pyc", line 304, in drawRect_
>   File "defconAppKit/controls/glyphView.pyc", line 328, in drawVerticalMetrics
>   File "defconAppKit/tools/drawing.pyc", line 207, in drawFontVerticalMetrics
> NameError: global name 'flipped' is not defined
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub <https://github.com/typesupply/defconAppKit/pull/15/files/00d45aecc6cccad9eead38af3ff333a02e0b2444#r70616761>, or mute the thread <https://github.com/notifications/unsubscribe/ABIp1lKBEnbvuzFbcUVHn-ZommyWsQSmks5qVNytgaJpZM4JLXWa>.
>